src/eachrow.jl

```diff
@@ -135,7 +136,7 @@ The modified `AbstractDataFrame`.
 ### Examples
 
 ```julia
-julia> using DataFrames, DataFramesMeta
+julia> using  DataFramesMeta
```

Comment:
why having this line at all? It seems clear that this should be loaded?

Reply:
while we are at it - at some point it would probably be good to turn on doctests

Reply:
yes. This would be a good check to have. Probably closer to 1.0.

Reply:
I'm not sure but at some point I decided to have every examples block add `using DataFramesMeta`. can change in a future PR.